packages/contracts-bedrock/src/L2/SuperchainERC20Factory.sol

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import { ISemver } from "src/universal/ISemver.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { IL2ToL2CrossDomainMessenger } from "src/L2/IL2ToL2CrossDomainMessenger.sol";
+import { ICrossDomainMessenger } from "@openzeppelin/contracts/vendor/optimism/ICrossDomainMessenger.sol";
```

Comment:
This import seems incorrect, the repo should have a way of fetching it from the repo itself instead of oz's package

Reply:
The thing is that there is no interface for `CrossDomainMessenger`, should we add one?

Tried using the contract but it fixed at 0.8.15 and was having `Found incompatible Solidity versions` errors